FirebaseCore/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Firebase 8.0.0
 - [removed] The deprecated Firebase InstanceID has been removed. Use Firebase Installations to manage
   app instance and use Firebase Messaging to manage FCM registration token instead. (#7970)
+- [changed] The experimental Carthage distribution is temporarily discontinued pending integration
+  with the pending Carthage 0.38.0 release with support for binary xcframeworks. In the meantime,
```

Comment:
Do you want to add a link to the PR for Carthage? https://github.com/Carthage/Carthage/pull/3152 I'm good either way.